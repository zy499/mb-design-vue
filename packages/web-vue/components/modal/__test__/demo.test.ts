import demoTest from '../../../scripts/demo-test';

demoTest('modal');
