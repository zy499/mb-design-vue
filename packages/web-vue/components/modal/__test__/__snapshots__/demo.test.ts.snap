// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<modal> demo: render [async] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Modal
</button>
<!---->"
`;

exports[`<modal> demo: render [basic] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Modal
</button>
<!---->"
`;

exports[`<modal> demo: render [custom] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Modal
</button>
<!---->"
`;

exports[`<modal> demo: render [draggable] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Draggable Modal
</button>
<!---->"
`;

exports[`<modal> demo: render [form] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Form Modal
</button>
<!---->"
`;

exports[`<modal> demo: render [fullscreen] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Modal
</button>
<!---->"
`;

exports[`<modal> demo: render [function] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Modal
</button>"
`;

exports[`<modal> demo: render [notice] correctly 1`] = `
"<div class=\\"mb-space mb-space-horizontal mb-space-align-center\\">
  <div class=\\"mb-space-item\\" style=\\"margin-right: 8px;\\"><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
      <!--v-if-->Info
    </button></div>
  <div class=\\"mb-space-item\\" style=\\"margin-right: 8px;\\"><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-success\\" type=\\"button\\">
      <!--v-if-->Success
    </button></div>
  <div class=\\"mb-space-item\\" style=\\"margin-right: 8px;\\"><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-warning\\" type=\\"button\\">
      <!--v-if-->Warning
    </button></div>
  <div class=\\"mb-space-item\\"><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-danger\\" type=\\"button\\">
      <!--v-if-->Error
    </button></div>
</div>"
`;

exports[`<modal> demo: render [width] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Modal
</button>
<!---->"
`;
